### DISCUSSION TITLE
Database adding the time offset for the timezone twice?

### DISCUSSION THREAD

**Discussion by kumsumit** [ArcadeData/arcadedb#1671] [Show and tell]
I have defined my property as follows
`CREATE PROPERTY Transaction.occurredAt DATETIME (readonly, default sysdate('YYYY-MM-DD HH:MM:SS'))`
When I insert the data, the occurredAt column has the value added the current time on my laptop. 
I am using the database as embedded
I have not done any configuration regarding the timezone, everything is as it was by default
You can clearly see the difference is approx +5:30 
(approx - because of the time taken by me to take screenshot may be 1 or 2 seconds)

`Highlighted`
![img1](https://github.com/user-attachments/assets/48470171-5175-49a5-9249-50bced26698a)

`Original`
![image](https://github.com/user-attachments/assets/ef8ff3cf-cc68-4214-af7c-e7d4938267c8)

**Comment by lvca** [comment]
Could you please take a screenshot of the timezone under the server settings (you can find it in Studio).

**Reply by kumsumit** [reply]
I didn't find it in the studio ( in the web version). Can you please guide me where I can find it out.

https://github.com/user-attachments/assets/6f3f7cd3-0b92-48e2-82d4-d9f826b87bfc

**Comment by gramian** [comment]
It may be important to know for this issue if you are running via Docker / Podman or on bare metal. If you are running via Docker it may be that the container does or can not use the host time zone. To test this pass the environment variabe `TZ` with your timezone, ie `TZ=CET` for central europe, and see if this fixes it.
